app/views/pending_loans/show.html.erb

```diff
@@ -16,8 +16,8 @@
     <% else %>
       <% @pending_loans.each do |project, loan_amount| %>
         <tr id=<%= "project_#{project.id}" %>>
-          <td><%= project.tenant.organization %></td>
-          <td><%= link_to project.title, project_path(project) %></td>
+          <td><%= project.tenant.slug %></td>
+          <td><%= link_to project.title, tenant_project_path(slug: project.tenant.organization, id: project.id) %></td>
```

Comment:
This is the one that needed slug: project.tenant.slug, since computers don't like "Save the children"